packages/common/src/HardhatConfig.js

```diff
@@ -1,8 +1,7 @@
 const { getNodeUrl, mnemonic } = require("./TruffleConfig");
 const path = require("path");
-const coreWkdir = path.dirname(require.resolve("@uma/core/package.json"));
 
-function getHardhatConfig(configOverrides) {
+function getHardhatConfig(configOverrides, truffleContextDir = "./") {
```

Comment:
nit: why not name this `workingDir`? Doesn't seem to have much to do with truffle right?